package.json

```diff
@@ -44,7 +44,7 @@
     "build:esm": "babel src --out-dir build/esm --extensions \".ts,.tsx\" --source-maps inline --ignore src/__tests__",
     "build:types-cjs": "tsc --emitDeclarationOnly --project tsconfig.build.json --outDir build/cjs && cp src/index.js.flow build/cjs",
     "build:types-esm": "tsc --emitDeclarationOnly --project tsconfig.build.json --outDir build/esm && cp src/index.js.flow build/esm",
-    "docs": "npx docsify-cli serve docs",
+    "docs": "docsify serve docs",
```

Comment:
Ah okay so the command should just be `docsify` and not `docsify-cli` if you've got it locally installed? I actually wanted to remove it from the dev dependencies but if this works then that's fine too.

Reply:
Yeah, actually it's added as a devDependency already. So I'll leave for you to decide, in that case

Reply:
https://github.com/atlassian-labs/react-resource-router/pull/13#issuecomment-638653545